src/musicbrainz/web/musicbrainzrecordingstask.cpp

```diff
@@ -172,24 +180,10 @@ void MusicBrainzRecordingsTask::doNetworkReplyFinished(
     // Continue with next recording id
     DEBUG_ASSERT(!m_queuedRecordingIds.isEmpty());
     emit currentRecordingFetched();
-    // Some of the tracks has loads of RecordingIds.
-    // This can cause hitting the rate limits of MusicBrainz.
-    // According to the MusicBrainz API Doc: https://musicbrainz.org/doc/MusicBrainz_API/Rate_Limiting
-    // The rate limit should be one query in a second.
-    // Related Bug: https://bugs.launchpad.net/mixxx/+bug/1983204
-    // In order to not hit the rate limits and respect their rate limiting rule.
-    // We are going to delay every request by one second.
-    if (m_timer.elapsed(true).toIntegerSeconds() >= 1) {
-        m_timer.restart(true);
-        slotStart(m_parentTimeoutMillis);
```

Comment:
Start here the QTimer with just 1 that it expires immediately

Reply:
Should QTimer would be 1 before the condition or before the condition so it can be 1 in all tasks?

Reply:
I do not understand the question.

Reply:
I think I understand it according to your requested change below. Thanks :)